1-exercises/B-while-loop/exercise.js

```diff
@@ -7,7 +7,16 @@
 
 function evenNumbers(n) {
     // TODO
-}
+     let i = 0;
+     let even = [];
+     while (i < 2 * n) {
+       even.push(i);
+       i += 2;
+     }
+     return even.toString();
+   }
+   console.log(evenNumbers());
```

Comment:
This logs out the array fine, looks good.
It's a minor point but the instruction is asking to log it out as a comma-separated string. Have a look at the array.join method.